bin/007

```diff
@@ -30,6 +30,7 @@ sub run_007($program, Str $backend is copy) {
     my $unexpanded = $backend eq "unexpanded-ast";
     my $ast = _007.parser(:$runtime).parse($program, :$unexpanded);
     %BACKENDS{$backend}($ast, $runtime);
+    exit($runtime.exit-code);
```

Comment:
Know nothing about Unix exit codes. I Glanced at a page - <http://tldp.org/LDP/abs/html/exitcodes.html> - and was thinking about something. Unsure if this is the place for questions, but for instance exit code 130 (CTRL+C) already works? A trivial observation, of course... so this would be equivalent with exit(130)? ...Could you say that this function basically reproduce the exit codes inherit in the System?

Reply:
I had no idea about exit code 130 for Ctrl+C. But trying this out on my system, from inside bash, it seems this is already the result. Not sure if that's thanks to bash or to Perl 6, but it seems to work as intended already.